R/ccTable.R

```diff
@@ -452,9 +452,17 @@ getEpisodePeriod <- function (e, unit="hours") {
     # The failure of POSIX conversion indicates that this episode is either 
     # anonymised or has a missing or incorrect value of discharge or admission
     # time. 
-    if (is.na(tadm) || is.na(tdisc))
-        period_length <- findMaxTime(e)
-    else {
+    if (is.na(tdisc)) {
```

Comment:
tadm can probably be NA in general. It is only a specific property of the dataset you have that it cannot.